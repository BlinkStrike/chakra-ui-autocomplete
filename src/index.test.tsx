import { CUIAutoComplete } from './index'

describe('CUIAutoComplete', () => {
  it('is truthy', () => {
    expect(CUIAutoComplete).toBeTruthy()
  })
})
