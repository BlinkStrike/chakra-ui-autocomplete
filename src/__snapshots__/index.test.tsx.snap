// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChakraUI AutoComplete  it renders 1`] = `
<div
  className="css-1010j8n"
>
  <div
    className="css-vy8xsp"
  >
    <h1
      className="chakra-heading css-h82jag"
    >
      Basic Example with Style Props
    </h1>
    <div
      className="chakra-stack css-wiimz6"
    >
      <label
        className="chakra-form__label css-1ig2arj"
        htmlFor="downshift-0-input"
        id="downshift-1-label"
      >
        Choose preferred work locations
      </label>
      <div
        className="chakra-stack css-l0vlzs"
      />
      <div
        aria-expanded={false}
        aria-haspopup="listbox"
        aria-owns="downshift-1-menu"
        className="chakra-stack css-101iv94"
        role="combobox"
      >
        <input
          aria-autocomplete="list"
          aria-controls="downshift-1-menu"
          aria-labelledby="downshift-1-label"
          autoComplete="off"
          className="chakra-input css-0"
          id="downshift-0-input"
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Type a Country"
          value=""
        />
        <button
          aria-label="toggle menu"
          className="chakra-button css-oqfhsh"
          id="downshift-1-toggle-button"
          onClick={[Function]}
          tabIndex={-1}
          type="button"
        >
           
          ↓
           
        </button>
      </div>
      <div
        className="css-vm1uz1"
      >
        <ul
          aria-labelledby="downshift-1-label"
          className="css-1jrokhb"
          id="downshift-1-menu"
          onMouseLeave={[Function]}
          role="listbox"
        />
      </div>
    </div>
    <div
      className="css-kkyj92"
    />
    <h1
      className="chakra-heading css-h82jag"
    >
      Basic Example with Custom Renderer
    </h1>
    <div
      className="chakra-stack css-wiimz6"
    >
      <label
        className="chakra-form__label css-1ig2arj"
        htmlFor="downshift-2-input"
        id="downshift-3-label"
      >
        Choose preferred work locations
      </label>
      <div
        className="chakra-stack css-l0vlzs"
      />
      <div
        aria-expanded={false}
        aria-haspopup="listbox"
        aria-owns="downshift-3-menu"
        className="chakra-stack css-101iv94"
        role="combobox"
      >
        <input
          aria-autocomplete="list"
          aria-controls="downshift-3-menu"
          aria-labelledby="downshift-3-label"
          autoComplete="off"
          className="chakra-input css-0"
          id="downshift-2-input"
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Type a Country"
          value=""
        />
        <button
          aria-label="toggle menu"
          className="chakra-button css-oqfhsh"
          id="downshift-3-toggle-button"
          onClick={[Function]}
          tabIndex={-1}
          type="button"
        >
           
          ↓
           
        </button>
      </div>
      <div
        className="css-vm1uz1"
      >
        <ul
          aria-labelledby="downshift-3-label"
          className="css-1jrokhb"
          id="downshift-3-menu"
          onMouseLeave={[Function]}
          role="listbox"
        />
      </div>
    </div>
  </div>
</div>
`;
